NoSqlAssignment/blog_schema.doc

```diff
@@ -0,0 +1,88 @@
+
+Following schema is for collection posts:
+
+{
+	_id:post_id ,
+	post_heading : "Title of the post" ,
+	
+	post_body : "Contains description of post in form of text, images, html files etc" ,
+	
+	post_made_by : "Username of the post" ,
+	
+	post_link : "Contains post link to be shown" ,
+	
+	tags : [tag1,tag2] ,
+	
+	post_created : 'date of post' ,
+	
+	comment : [
```

Comment:
If the values is an array, the key name should be "comments" not singular "comment"